app/models/breakeven_table_question.rb

```diff
@@ -133,10 +133,10 @@ def calculate_line_item_totals(total_key)
       next unless product.present? && product[:name].present?
       {
         name: product[:name],
-        quantity: product.fetch(:quantity, 0),
-        amount: product.fetch(total_key, 0),
+        quantity: product.fetch(:quantity, 0).to_f,
```

Comment:
I fixed a couple more issues with `breakeven_table_question` and added some specs to cover it.